features/step_definitions/oauth.rb

```diff
@@ -14,5 +14,6 @@
     opts = {resource: 'oauth', resource_name: name, p: patch_json, type: 'merge' }
     @result = _admin.cli_exec(:patch, **opts)
     raise "Cannot restore OAuth: #{name}" unless @result[:success]
+    sleep 60
```

Comment:
Can we instead of hardcode waiting, check when it becomes usable? That means repeat an operation that will succeed only after the CRD works properly? WDYT?

Reply:
hah, didn't see Peter's comment